library/ajax/left_nav_encounter_ajax.php

```diff
@@ -77,30 +77,14 @@
 if ($createvisit) {
   // Write JavaScript to open the selected encounter as the active encounter.
   // Logic cloned from encounters.php.
-  if ($GLOBALS['concurrent_layout']) {
 ?>
 top.left_nav.setEncounter('<?php echo $today; ?>', enc, 'RBot');
-top.left_nav.setRadio('RBot', 'enc');
 top.left_nav.loadFrame2('enc2', 'RBot', 'patient_file/encounter/encounter_top.php?set_encounter=' + enc);
```

Comment:
Is RBot etc... still in use everywhere?  Forgive me, I am swapping my projects around too much to keep my brain in gear.